.gitignore

```diff
@@ -0,0 +1,4 @@
+
+test.md
+**/node_modules
+*.md
```

Comment:
- [ OPTIONAL] I can't see the meaning of this file here can you please remove it for good practice to make your work professional :wink: